nncf/quantization/algorithms/fast_bias_correction/backend.py

```diff
@@ -31,13 +31,6 @@
 
 
 class FastBiasCorrectionAlgoBackend(ABC):
```

Comment:
Please update the type hints to take into account the migration from the tensor framework to nncf.Tensor.